### PR TITLE
Loader and Date picker using antd 

### PR DESCRIPTION
**Pull request by rakhimundhada15**

### CODE REVIEW

File: ats-web/src/components/shared/datePicker.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import PropTypes from "prop-types";
+import { DatePicker } from 'antd';
+
+function DateTimePicker(props) {
+    const [startDate, setStartDate] = useState(null);
```
Comment (K-Nihar) [inline ats-web/src/components/shared/datePicker.js]: Rename startDate.

File: ats-web/src/components/shared/datePicker.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    const handleOnOk = (selectedDate) => {
+        if (selectedDate) {
+            props.onChange(selectedDate._d)
```
Comment (K-Nihar) [inline ats-web/src/components/shared/datePicker.js]: send whole object instead of just date

Comment (rakhimundhada15) [reply ats-web/src/components/shared/datePicker.js]: Send complet date object

File: ats-web/src/components/shared/datePicker.js (hunk 1 above)
Comment (rakhimundhada15) [reply ats-web/src/components/shared/datePicker.js]: Rename  startdate -> date

**Pull request merged**